systems/framework/BUILD.bazel

```diff
@@ -48,7 +48,6 @@ cc_test(
 cc_library(
```

Comment:
Entire binary should be deleted.

Reply:
What do you mean by deleting the binary?

Reply:
My mistake--I misinterpreted something.